api/v4/source/introduction.yaml

```diff
@@ -598,6 +598,7 @@ x-tagGroups:
       - roles
       - schemes
       - integration_actions
+      - remote clusters
```

Comment:
Elsewhere `Remote Clusters` was renamed `Secure Connections`.  Not sure if that is still a thing.  May want to check with a PM.

Reply:
Good point, on the prototypes of the system console they're referred to as "Connected Organisations", so we probably want to get some consistency in the docs